utilities/src/main/java/io/syndesis/qe/utils/OpenShiftUtils.java

```diff
@@ -392,4 +394,29 @@ public static boolean hasPodIssuesPullingImage(Pod pod) {
                 && ("ImagePullBackOff".equals(status.getState().getWaiting().getReason()) ||
                 "ErrImagePull".equals(status.getState().getWaiting().getReason())));
     }
+
+    public static void waitTillPodIsReloaded(String podPartialName, int timeoutInSeconds) {
+        log.info(String.format("Waiting until %s pod is reloaded", podPartialName));
+
+        final int currentNr = OpenShiftUtils.extractPodSequenceNr(OpenShiftUtils.getPodByPartialName(podPartialName).get());
+        TestUtils.waitFor(() -> OpenShiftUtils.podExists(
+            p -> p.getMetadata().getName().contains(podPartialName),
+            p -> !StringUtils.containsAny(p.getMetadata().getName(), "build", "deploy"),
+            p -> OpenShiftUtils.extractPodSequenceNr(p) > currentNr),
+            5, timeoutInSeconds,
+            "The pod " + podPartialName + " was not reloaded in time.");
+    }
+
+    public static boolean dcContainsEnv(String dcName, String envName) {
+        return OpenShiftUtils.getInstance().getDeploymentConfig(dcName)
+            .getSpec().getTemplate().getSpec().getContainers().get(0).getEnv()
+            .stream().anyMatch(envVar -> envVar.getName().equals(envName));
+    }
+
+    public static boolean envInDcContainsValue(String dcName, String envName, String envValue) {
+        EnvVar envVar1 = OpenShiftUtils.getInstance().getDeploymentConfig(dcName)
+            .getSpec().getTemplate().getSpec().getContainers().get(0).getEnv().stream().filter(envVar -> envVar.getName().equals(envName)).findAny()
+            .get();
+        return envVar1.getValue().equals(envValue);
```

Comment:
if, for some reason, the `getValue()` will return `null`, it will fail with NPE, so it's better to do: `envValue.equals(envVar.getValue())`